Dockerfile.xaodTransformer

```diff
@@ -31,6 +31,9 @@ COPY bashrc /home/atlas/.bashrc
 COPY bashrc /home/atlas/.bash_profile
 COPY requirements.txt .
 RUN source /home/atlas/release_setup.sh; \
+    python2 -m pip install --user safety==1.8.7
```

Comment:
Is the version of safety different here because of python2 vs python3?

Reply:
Exactly; I put in the latest available version of safety for Python 2